JavaScript/Task3/CanYouKeepASecret.js

```diff
@@ -0,0 +1,12 @@
+//http://www.codewars.com/kata/can-you-keep-a-secret
+
+function createSecretHolder(secret) {
+  return {
+    getSecret: function() {
```

Comment:
Use anonymous functions

Reply:
Can you please tell me how to do it? I'm a not able to understand how to use anonymous function in case.

Reply:
getSecret : () => {}

Reply:
done